### ISSUE TITLE
Possibility of updating to use official Speedtest CLI?

### ISSUE THREAD

**Issue by Jiriteach** [roest01/docker-speedtest-analyser#36]
Hi - I've been using this and its worked well but due to the inconsistencies in speed with the speedtest-cli library you are using, I am having to look for an alternative.

Example - I have a 1gig connection. My speed max's at 500 through the current library regardless of server that I try.

Speedtest.net . have released an official speedtest cli library here - https://www.speedtest.net/apps/cli
Any chance of an option of using this? I've tried this via cmd line and it works as expected.

Would be awesome to have an updated Docker image that executes this instead.

Thanks

**Comment by roest01** [comment]
Hey :)
Can you do an pull request and implement the new official cli?
It's a python script to modify....

**Comment by roest01** [comment]
btw i am a little jealous of your 1gb connection lol

**Comment by Jiriteach** [comment]
> Hey :)
> Can you do an pull request and implement the new official cli?
> It's a python script to modify....

Yup - Will try and sort something this weekend. Wasn't sure if you had already tried or not :)

**Comment by dra2885** [comment]
I’d love to see this enhancement as well. My connection appears to max out at around 600 Mbps, but I suspect it’s due to the limitation noted above.

**Comment by dnagl** [comment]
PR #40 should resolve this issue.
